### DISCUSSION TITLE
"Vercel for Open Source" - offer some special plans for opensource projects?

### DISCUSSION THREAD

**Discussion by mathieucaroff** [vercel/vercel#4695] [General] [answered]
Have you been thinking of helping out Open Source projects and communities by providing free plans for them?

I'm thinking of something like what [Algolia](https://www.algolia.com/for-open-source) did. Vercel being partially Open Source makes me think that this is something you might like to do.

**Comment by mcsdevv** [comment, marked as answer]
Hello @mathieucaroff, we definitely do this already 😀

We've updated our pricing page to mention it but are planning to make this a lot more visible.

If you have a project you'd like to us to look at sponsoring, please give us a shout on the [support channel](vercel.com/support/request) and we'll see what we can do!
